### PR TITLE
Modifications to admin site and security audits

### PR DESCRIPTION
**Pull request by istanfield09**
Adds sufficient session checking for admin sessions.
Switches admin site to admin site 1 rather than 2.
Misc. bug fixes and changes to admin panel.

### CODE REVIEW

File: public_html/AdminSite2/components/loginStuff/checkSession.php [hunk 1]
```diff
@@ -4,6 +4,6 @@
   	session_start();
 
 	if(!(isset($_SESSION['username'])) || $_SESSION['username'] == "") {
-    header( 'Location: /AdminSite2/loginPage.php' ) ;
+    header( 'Location: /AdminSite/loginPage.php' ) ;
```
Comment (istanfield09) [inline public_html/AdminSite2/components/loginStuff/checkSession.php]: This doesn't need to be changed.

File: App/Lib/xmlGenerator/reuse_database.xml [hunk 2]
```diff
@@ -1,2 +1,2 @@
...
...</subcategory><subcategory>Fencing</subcategory><subcategory>Garden Pots</subcategory><subcategory>Garden Tools</subcategory><subcategory>Hand Clippers</subcategory><subcategory>Hoses</subcategory><subcategory>Lawn Furniture</subcategory><subcategory>Livestock Supplies</subcategory><subcategory>Loppers</subcategory><subcategory>Mowers</subcategory><subcategory>Seeders</subcategory><subcategory>Soil Amendment</subcategory><subcategory>Sprinklers</subcategory><subcategory>Wheelbarrows</subcategory></subcategory_list></category><category><name>Medical Supplies</name><subcategory_list><subcategory>Adult Diapers</subcategory><subcategory>Blood Pressure Monitors</subcategory><subcategory>Canes</subcategory><subcategory>Crutches</subcategory><subcategory>Eye Glasses</subcategory><subcategory>Glucose Meters</subcategory><subcategory>Hearing Aids</subcategory><subcategory>Hospital Beds</subcategory><subcategory>Reach Extenders</subcategory><subcategory>Shower Chairs</subcategory><subcategory>Walkers</subcategory><subcategory>Wheelchairs</subcategory></subcategory_list></category><category><name>Office Equipment</name><subcategory_list><subcategory>Calculators</subcategory><subcategory>Compu
```
Comment (istanfield09) [inline App/Lib/xmlGenerator/reuse_database.xml]: Need to undo this.

**Comment by japfohl** [comment]
Hey @istanfield09 whats the status of this?  Anything I can do to help?  It'd be nice to get this plugged into the codebase prior to submitting the midpoint check but also understandable if its not read / too much work.

File: public_html/js/searchBusFunct.js [hunk 3]
```diff
@@ -90,7 +90,11 @@ function allBusinesses() {
             $('#EditData2').empty();
             var row = '<tr><th><span class="locked">' + 'Name' + '</span></th><th><span class="locked">' + 'Address' + '</span></th><th><span class="locked">' + 'Modify' + '</span></th><th><span class="locked">' + 'Delete' + '</span></th></tr>';
             for (var i = 0; i < data.length; i++) {
-                row += '<tr><td>' + data[i].name + '</td><td>' + data[i].address_line_1 + '</td><td>' + "<button value='" + data[i].name + "' type=submit id=edit onclick='editBusinessHelper(this.value); return false;'>" + 'Edit' + "</button>" + '</td><td>' + '<input type= hidden id= delete value=' + data[i].id + '><input class= del type= submit value= Delete id= ' + data[i].id + ' onclick=delBusiness(this.id)>' + '</td>' + '</tr>';
+                row += '<tr><td>' + data[i].name + '</td><td>' + data[i].address_line_1 +
```
Comment (japfohl) [inline public_html/js/searchBusFunct.js]: This whole section is ugly... merge anyways but its super difficult to read.  (won't be an issue once we move rendering to PHP).

File: public_html/js/searchBusFunct.js [hunk 4]
```diff
@@ -163,8 +167,9 @@ function editBusiness() {
     // $('#searchForm').hide();
     clearAll();
     $('#editFields').show();
-    var nameOfBizToEdit =  $('#searchName').val();
-
+    var nameOfBizToEdit = $('#searchName').val();
+    console.log(nameOfBizToEdit);
```
Comment (japfohl) [inline public_html/js/searchBusFunct.js]: Do we want to leave the `console.log()`s in in production?

File: src/routes/AdminRoutes.php [hunk 5]
```diff
@@ -1,4 +1,15 @@
 <?php
+
+// Helper used to safely fetch POST values. This should be placed
+// somewhere better eventually. <~ TODO
+function fetch_val($key, $array, $default = 'undefined') {
```
Comment (japfohl) [inline src/routes/AdminRoutes.php]: Nice!

File: test/tests/AdminRoutesTest.php [hunk 6]
```diff
@@ -0,0 +1,38 @@
...
+
+    public function testAddBusinessDocRoute()
+    {
+        $response = $this->client->request('POST', '/RUapi/addBusinessDoc', [
+            'json' => ['doc_name' => 'my_special_document', 
+                       'doc_url' => 'https://www.google.com',
+                       'business_id' => 1]
+        ]);
+
+        $this->assertEquals(200, $response->getStatusCode());
```
Comment (japfohl) [inline test/tests/AdminRoutesTest.php]: Are you going to expand the scope of this test?  Like, actually test that the DB was modified, etc...?

File: public_html/js/searchBusFunct.js (hunk 4 above)
Comment (istanfield09) [reply public_html/js/searchBusFunct.js]: I do not.

**Pull request merged**